src/commands/deploy/nestjs/deploy.ts

```diff
@@ -133,8 +137,8 @@ async function deployFunction(
             path: cwdRelative,
             language: {
                 name: Language.js,
-                runtime: "nodejs20.x",
-                architecture: "x86_64",
+                runtime: DEFAULT_NODE_RUNTIME,
+                architecture: DEFAULT_ARCHITECTURE,
```

Comment:
What happens if I already set the desired version in the YAML file and it's different from the default? 

```suggestion
                runtime: (config.nestjs?.runtime as NodeRuntime) || DEFAULT_NODE_RUNTIME,
                architecture: (config.nestjs?.architecture as Architecture) || DEFAULT_ARCHITECTURE,
```

same suggestion for the rest of the ssr type

Reply:
And one more thing, for this suggestion, you should extend the SSR schema with the architecture type.

Reply:
`architecture` is not yet a field that's used, I don't want to make changes to promote it to our users.